db/1.9/modules/ROOT/pages/database-stored-procedure.adoc

```diff
@@ -0,0 +1,209 @@
+
+. In your Studio flow, select the *Stored procedure* operation.
+. In the operation configuration screen, set the *SQL Query Text* field to the query to execute, for example:
++
+`{ call multiplyInts(:int1, :int2, :result1, :int3, :result2) }`
++
+. Set the *Input parameters* field to `
++
+[source,xml,linenums]
+----
+{
+    'int1' : 3,
+    'int2' : 4,
+    'int3' : 5
+}
+----
++
+[start=4]
+. Set the *Output parameters* field to `Edit inline`.
+. Click the plus sign (*+*) to add a new value.
+. In the new window, set the *Key* field to `result1` and the *Type* field to `INTEGER`.
+. Repeat the previous three steps to add new values, set the *Key* field to `result2` and the *Type* field to `INTEGER`.
+. Create another value again by setting the *Key* field to `myInt` and the *Type* field to `INTEGER`.
+
+The following screenshot shows the configuration:
+
+.Stored procedure with Input and Output parameters configuration
+image::database-stored-example-3.png[Stored procedure with Input and Output parameters configuration]
+
+In the XML editor, the `<db:input-parameters>` and `<db:output-parameters>` configuration looks like this:
+
+[source,xml,linenums]
+----
+<db:stored-procedure config-ref="dbConfig">
+    <db:sql>{ call multiplyInts(:int1, :int2, :result1, :int3, :result2) }</db:sql>
+    <db:input-parameters>
+        #[{
+            'int1' : 3,
+            'int2' : 4,
+            'int3' : 5
+        }]
+    </db:input-parameters>
+    <db:output-parameters>
+        <db:output-parameter key="result1" type="INTEGER"/>
+        <db:output-parameter key="result2" type="INTEGER"/>
+        <db:output-parameter key="myInt" type="INTEGER"/>
+    </db:output-parameters>
+</db:stored-procedure>
+----
+
+== Callable Statements
+
+Database Connector supports the use of callable statements with the format:
+
+`{ call procedureName(:param1, :param2, ..., :paramN) }`
+
+The parameters matches positionally, `:paramN` matches the `Nth` parameter in the stored procedure declaration. Make sure that no whitespace is between the name of the stored procedure and the first parentheses.
+
+In the following example, you are using an Oracle database which was initialized with the creation of table named `SYSTEM.employees` and stored procedure named `createEmployee`:
+
+[source,xml,linenums]
+----
+    CREATE TABLE SYSTEM.employees(
+        employee_id INTEGER GENERATED BY DEFAULT AS IDENTITY,
+        employee_name VARCHAR2(100),
+        employee_age INTEGER,
+        employee_birthday TIMESTAMP,
+        PRIMARY KEY(employee_id)
+    );
+
+
+    CREATE PROCEDURE createEmployee(e_name VARCHAR2, e_age NUMBER, e_birth_date DATE) AS
+    BEGIN
+        INSERT INTO SYSTEM.employees(employee_name, employee_age, employee_birthday) VALUES(e_name, e_age, e_birth_date);
+    END;
+----
+
+Database Connector does not support named parameters in callable statements, so attempting to match parameters by name does not work, for example:
```

Comment:
This sentence is correct.
Here is another possible wording:
```suggestion
Because Database Connector does not support named parameters in callable statements, attempting to match parameters by name does not work, for example:
```